starter-apps/vue/vue-template/src/components/Heading.vue

```diff
@@ -0,0 +1,6 @@
+<script setup>
+const { tag } = defineProps(['tag'])
```

Comment:
We should also add the two `margin` props.

Reply:
Those two other props are optional which is why this didn't automatically include them -- but I can include that here